src/EFCore.Jet/Storage/Internal/JetDecimalTypeMapping.cs

```diff
@@ -77,7 +79,18 @@ protected override RelationalTypeMapping Clone(RelationalTypeMappingParameters p
         protected override void ConfigureParameter(DbParameter parameter)
         {
             base.ConfigureParameter(parameter);
-
+            //Decimals needs to be mapped to Numeric for Jet.
+            //Using Decimal is fine for OleDb but Odbc doesn't like it.
+            //Have to use Numeric for Odbc.
+            //Suspect this will also fix any formatting erros with , and . for decimal separator and space and , for digit separator
+            if (parameter is OdbcParameter odbcParameter)
+            {
+                odbcParameter.OdbcType = OdbcType.Numeric;
+            }
+            else if (parameter is OleDbParameter oleDbParameter)
+            {
+                oleDbParameter.OleDbType = OleDbType.Numeric;
+            }
```

Comment:
This (or maybe the entire type mapping) should be moved to the ODBC and OLE DB specific libraries, so that we can keep the main package clean.

Reply:
It should be possible to implement this without too much effort using `IRelationalTypeMappingSourcePlugin` for each of the provider specific libraries. No reflection neeeded.

Reply:
Sure. But you still need an entry point to be called so that it can register those plugins. Its not registered by default

Reply:
Nothing is registered by default. It's being done by `UseJet` / `JetCore` -> `JetOptionsExtensions.ApplyServices` -> `AddEntityFrameworkJet`.

Check which provider library is available and call into the library to let it register its services.

Reply:
Are you happy with using reflection? It's the simplest solution